index.html

```diff
@@ -56,29 +51,31 @@
       </p>
       <img class="Ellipse12" src="/images/Ellipse12.png" alt="shapes"/>
       <img class="Ellipse13" src="/images/Ellipse13.png" alt="shapes"/>
```

Comment:
- [ ] As the previous reviewer mentioned these images should be in the background and not as `img` elements, I saw that you already added a background, good job that was good, the issue here is that you exported shapes instead of the whole background image from the Figma template, I tried selecting for you the element in the Figma template, just go here to [this link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) and export the image (the element is called `header-shapes-mobile` and it's a collection of all the shapes). 😁 After you export it, kindly remove these elements from here, then go to your CSS file and replace the background image you have there with the new one, and that should be it! 😌